JSONRPC/JSONRPC.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.IO;
+using System.Text;
+
+using Newtonsoft.Json.Linq;
+
+public class JSONRPC {
+	private string _endpoint;
+
+	// Basic authentication
+	private string _username;
+	private string _password;
```

Comment:
It's still in debate for the style guide.  The single main reason for this is the use of properties.  For Zappallas we are trying lowercase private and public with Uppercase properties.  This is going OK .. but as expected when you have 

public string type;
public string Type { get { return type; } set { OnTypeSet.Invoke(); type = value; }

or even

public void SetType(string type) {  OnTypeSet.Invoke(); this.type = type; }

We have already had cases of people using the public string type to set the parameter.  So it relies heavily on code review (This is my single biggest grip with Unity still)